### PR TITLE
Adds more documentation.

### PR DESCRIPTION
**Pull request by vpetersson**
@ryanteck Please confirm that the below is all correct.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,73 @@
...
+## Gateway Config
 
-## helium-miner
+Repo: [github.com/NebraLtd/hm-config](https://github.com/NebraLtd/hm-config)
 
-The Helium Miner with the required configuration files added.
+This container is responsible for the device onboarding and provides the Bluetooth LE to allow the hotspot to be configured via the Helium App.
 
-https://github.com/NebraLtd/hm-miner
+## Helium Miner
```
Comment (ryanteck) [inline README.md:27]: This was a deliberate choice to have it written as helium-miner

File: README.md [hunk 2]
```diff
@@ -1,58 +1,73 @@
...
+There are currently six different services running within this device, which are all outlined below.
 
-https://github.com/NebraLtd/hm-diag
+## Diagnostics
 
-## packet-forwarder
+Repo: [github.com/NebraLtd/hm-diag](https://github.com/NebraLtd/hm-diag)
 
-The container that has the code that configures the packet forwarder's region and starts the radio module.
+The diagnostics container is designed for local troubleshooting. It runs a local web server that displays various diagnostics data.
```
Comment (ryanteck) [inline README.md:13]: Not documenting that this is responsible for manufacturing tools is a pretty big omission.

File: README.md [hunk 3]
```diff
@@ -1,58 +1,73 @@
...
+This container is responsible for configuring packet forwarder's region and starts the radio module.
 
-https://github.com/NebraLtd/hm-config
+## Gateway Config
 
-## helium-miner
+Repo: [github.com/NebraLtd/hm-config](https://github.com/NebraLtd/hm-config)
 
-The Helium Miner with the required configuration files added.
+This container is responsible for the device onboarding and provides the Bluetooth LE to allow the hotspot to be configured via the Helium App.
```
Comment (ryanteck) [inline README.md:25]: Well it's partially responsible, as much of the onboarding side is done on Helium side. This is as much for wi-fi configuration and following through the steps.

File: README.md (hunk 1 above)
Comment (ryanteck) [reply README.md:27]: This applies to all other headings that have been changed too.

**Review by ryanteck — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (shawaj) [reply README.md:27]: Could have as Helium Miner and then below have...

Repo: link
Container: helium-miner

Or something to that effect

**Pull request merged**